sklearn/neighbors/_classification.py

```diff
@@ -401,22 +422,25 @@ def __init__(self, radius=1.0, *, weights='uniform',
         self.outlier_label = outlier_label
 
     def fit(self, X, y):
-        """Fit the model using X as training data and y as target values
+        """Fit the radius neighbors classifier using `X` as training data and
+        `y` as target values.
```

Comment:
This should be a 1 line summary: https://numpydoc.readthedocs.io/en/latest/format.html#sections
Also apparently that's does not use variable names, but I think the previous version was fine.

I think the previous version was fine.

Reply:
Thanks @rth for the suggestion!

I have kept the 1-line summary and remove the variable names.